04-mpu6050/module/mpu6050.c

```diff
@@ -77,20 +69,64 @@ static const struct i2c_device_id mpu6050_idtable[] = {
 	{ "mpu6050", 0 },
 	{ }
 };
+MODULE_DEVICE_TABLE(i2c, mpu6050_idtable);
 
 static struct i2c_driver mpu6050_i2c_driver = {
 	.driver = {
 		.name = "con_mpu6050",
 	},
+	
 	.probe = mpu6050_probe,
 	.remove = mpu6050_remove,
 	.id_table = mpu6050_idtable,
 };
 
+static int mpu6050_read_data(void)
+{
+	int ret;
+	int temp;
+
+	if (device_data.client == 0) {
+		ret = -ENODEV;
+		goto err;
+	}
+
+	temp = (s16)((u16)i2c_smbus_read_word_swapped(device_data.client, REG_TEMP_OUT_H));
+	device_data.temperature = ((((temp + 12420 + 170) * ONE_TEMP_POINT) / 340) * 9) / 5 + 32000;
+
+	pr_info("cpu6050: temperature = %d.%d\n", device_data.temperature / ONE_TEMP_POINT, device_data.temperature % ONE_TEMP_POINT);
```

Comment:
This output will give output without leading 0 for second value.  Temperature 10.005 is not the same as 10.5